Source/WebCore/rendering/RenderFrame.cpp

```diff
@@ -50,7 +51,8 @@ HTMLFrameElement& RenderFrame::frameElement() const
 
 FrameEdgeInfo RenderFrame::edgeInfo() const
 {
-    return FrameEdgeInfo(frameElement().noResize(), frameElement().hasFrameBorder());
+    CheckedRef frameElement = this->frameElement();
```

Comment:
Is there a reason to use CheckedRef over just `Ref` ? elements can be ref counted